src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -76,6 +93,18 @@
                 :message (str "Invalid level: " level)}))
     :info))
 
+(defn get-service-status-version
+  "Given a params map from a request, get out the service status version and
+   check whether it is valid. If not, throw an error."
+  [params]
+  (when-let [level (params :service-status-version)]
+    (let [parsed-level (ks/parse-int level)]
+      (if (nil? parsed-level)
```

Comment:
I think this could be simplified by doing `(if-let [parsed-level (ks/parse-int level)]`